mingw-w64-cross-clang/PKGBUILD

```diff
@@ -56,6 +56,11 @@ build() {
     # Force win32 threads for libc++abi
     export CXXFLAGS+=" -D_LIBCPP_HAS_THREAD_API_WIN32"
     COMMON_CMAKE_ARGS=(-G'Unix Makefiles'
+      -DCMAKE_AR="$(which llvm-ar)"
+      -DCMAKE_AS="clang"
```

Comment:
Sorry, I misread CMake docs, there is no `CMAKE_AS` so it doesn't do anything.

Reply:
I just discovered that myself